packages/compiler-ssr/src/transforms/ssrTransformSlotOutlet.ts

```diff
@@ -37,16 +38,18 @@ export const ssrTransformSlotOutlet: NodeTransform = (node, context) => {
 
     let method = SSR_RENDER_SLOT
 
-    // #3989
+    // #3989, #9933
     // check if this is a single slot inside a transition wrapper - since
-    // transition will unwrap the slot fragment into a single vnode at runtime,
+    // transition/transition-group will unwrap the slot fragment and render its children at runtime,
     // we need to avoid rendering the slot as a fragment.
     const parent = context.parent
     if (
       parent &&
       parent.type === NodeTypes.ELEMENT &&
       parent.tagType === ElementTypes.COMPONENT &&
-      resolveComponentType(parent, context, true) === TRANSITION &&
+      [TRANSITION, TRANSITION_GROUP].includes(
+        resolveComponentType(parent, context, true) as symbol,
+      ) &&
```

Comment:
I wonder instead of using includes wouldn't be better to do an `||` condition, seems unnecessary to create an array and then iterate it for 2 cases.

Reply:
done!